crates/anvil/src/eth/miner.rs

```diff
@@ -131,6 +145,23 @@ impl MiningMode {
             Self::None => Poll::Pending,
             Self::Auto(miner) => miner.poll(pool, cx),
             Self::FixedBlockTime(miner) => miner.poll(pool, cx),
+            Self::Mixed(auto, fixed) => {
+                let auto_txs = auto.poll(pool, cx);
+                let fixed_txs = fixed.poll(pool, cx);
+
+                match (auto_txs, fixed_txs) {
+                    // Both auto and fixed transactions are ready, combine them
+                    (Poll::Ready(mut auto_txs), Poll::Ready(mut fixed_txs)) => {
+                        auto_txs.append(&mut fixed_txs);
```

Comment:
there might be the case here where both include the the same txs so we need to make the set unique, basically only append if the not included in auto_txs